### ISSUE TITLE
compiling on win64 fails

### ISSUE THREAD

**Issue by RasPiTool** [CATIA-Systems/FMPy#60]
Hello,

I have a problem. :D 
I tried to compile a FMU and I get some error:
```
(base) C:\Users\mesau\Desktop\Masterarbeit\3-Software\8-Python\3-RasPi-PC>fmpy compile CoupledClutches.fmu
Compiling CoupledClutches.dll...
Das System kann den angegebenen Pfad nicht finden.
C:\Users\mesau\AppData\Local\Temp\tmpiba7p9cs\sources
call "%VS140COMNTOOLS%\..\..\VC\vcvarsall.bat" x86_amd64 && cl /LD /I. /I"C:\Users\mesau\Anaconda3\lib\site-packages\fmpy\c-code\fmi2" /DCO_SIMULATION /DFMI2_FUNCTION_PREFIX=CoupledClutches_ /FeCoupledClutches shlwapi.lib all.c C:\Users\mesau\Anaconda3\lib\site-packages\fmpy\c-code\fmi2\fmi2_wrapper.c
Traceback (most recent call last):
  File "C:\Users\mesau\Anaconda3\Scripts\fmpy-script.py", line 5, in <module>
    sys.exit(fmpy.command_line.main())
  File "C:\Users\mesau\Anaconda3\lib\site-packages\fmpy\command_line.py", line 77, in main
    compile_platform_binary(args.fmu_filename)
  File "C:\Users\mesau\Anaconda3\lib\site-packages\fmpy\util.py", line 631, in compile_platform_binary
    binary = compile_dll(model_description=md, sources_dir=os.path.join(unzipdir, 'sources'))
  File "C:\Users\mesau\Anaconda3\lib\site-packages\fmpy\util.py", line 609, in compile_dll
    raise Exception('Failed to compile shared library')
Exception: Failed to compile shared library
```
The system can not find the path.
In the method
`compile_dll(model_description, sources_dir, compiler=None) 
`
I add
`print("Path: ", dll_path) 
`
before this if statement
```
if status != 0 or not os.path.isfile(dll_path):
        raise Exception('Failed to compile shared library')

```
and get this path 
`C:\Users\mesau\AppData\Local\Temp\tmphnlcx13t\sources\CoupledClutches.dll
`
and there isn't a CoupledClutches.dll file.

**Comment by t-sommer** [comment]
Does the FMU contain source code? If so, can you run the command

```
call "%VS140COMNTOOLS%\..\..\VC\vcvarsall.bat" x86_amd64 && cl /LD /I. /I"C:\Users\mesau\Anaconda3\lib\site-packages\fmpy\c-code\fmi2" /DCO_SIMULATION /DFMI2_FUNCTION_PREFIX=CoupledClutches_ /FeCoupledClutches shlwapi.lib all.c C:\Users\mesau\Anaconda3\lib\site-packages\fmpy\c-code\fmi2\fmi2_wrapper.c
```
to get more detailed output?

**Comment by RasPiTool** [comment]
The FMU contains sources code. Here is the FMU
[CoupledClutches.zip](https://github.com/CATIA-Systems/FMPy/files/2607415/CoupledClutches.zip)
This is output for the command:
```
(base) C:\Users\mesau\Desktop\Masterarbeit\3-Software\8-Python\3-RasPi-PC>call "%VS140COMNTOOLS%\..\..\VC\vcvarsall.bat" x86_amd64 && cl /LD /I. /I"C:\Users\mesau\Anaconda3\lib\site-packages\fmpy\c-code\fmi2" /DCO_SIMULATION /DFMI2_FUNCTION_PREFIX=CoupledClutches_ /FeCoupledClutches shlwapi.lib all.c C:\Users\mesau\Anaconda3\lib\site-packages\fmpy\c-code\fmi2\fmi2_wrapper.c
Das System kann den angegebenen Pfad nicht finden.
```
The system can not find the path.

**Comment by t-sommer** [comment]
Do you have Visual Studio 14 (2015) installed on your machine?

**Comment by RasPiTool** [comment]
No, I have Visual Studio (2017)

**Comment by t-sommer** [comment]
Visual Studio 2017 is currently not supported

**Comment by chrbertsch** [comment]
Is there a list of supported compilers somewhere in the documentation?

**Comment by t-sommer** [comment]
All Visual Studio versions should work as long as they can compile the FMU's sources. The problems seems to be that Microsoft has changed the way the environment variables are set in 2017.

**Comment by t-sommer** [comment]
Resolved with 9b35f0f26f9696d1462cbc1f69942874a3e3b5ba
